source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx

```diff
@@ -221,6 +222,11 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 			const result = await cloneRepository(args)
 			// not blocking the execution by using the callback pattern
 			// the user does not need to wait for the response
+			// checks whether the gitOrigin corresponds to the pattern.
+			const gitOrigin = originChecked(await getGitOrigin(args))
+			telemetryBrowser.group("repository", gitOrigin, {
+				name: gitOrigin,
```

Comment:
why does the name need to be set?

Reply:
You must set both to create a new group/repository. 
Otherwise, Posthog will not create a new group.